darwin/importer/formats/pascalvoc.py

```diff
@@ -8,6 +8,23 @@
 def parse_file(path: Path) -> Optional[dt.AnnotationFile]:
     """
     Parses the given pascalvoc file and maybe returns the corresponding annotation.
+    The file must have the following structure:
```

Comment:
I think the <annotation> tag is missing. The [official website](http://host.robots.ox.ac.uk/pascal/VOC/voc2007/) is down, I would like to add a link to it in the doc string

Reply:
https://gist.github.com/Prasad9/30900b0ef1375cc7385f4d85135fdb44